testsuite/helloworld/tests/cpp_badalloc/test.yml

```diff
@@ -6,3 +6,4 @@ expect: [RTE, MLE]
 feedback:
 - std::bad_alloc
 - std::bad_array_new_length
+- ''
```

Comment:
Can we separate this into `test.linux.yml` and `test.freebsd.yml`?

Reply:
Done.